extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -125,12 +131,16 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 
 		adElement.setPageLevelParams(pageLevelParams);
 		if (!slot) {
-			if (sizes) {
-				slot = window.googletag.defineSlot(adElement.getSlotPath(), sizes, slotId);
-			} else {
-				slot = window.googletag.defineOutOfPageSlot(adElement.getSlotPath(), slotId);
-			}
+			slot = sizes ?
+				window.googletag.defineSlot(adElement.getSlotPath(), sizes, slotId) :
+				window.googletag.defineOutOfPageSlot(adElement.getSlotPath(), slotId);
+
 			slot.addService(window.googletag.pubads());
+
+			if (pageFair && pageFair.isSlotRecoverable(adElement.getSlotName())) {
```

Comment:
Shouldn't it be:
`pageFairRecovery && pageFair.isSlotRecoverable(adElement.getSlotName())`?